src/EnergyPlus/BranchInputManager.cc

```diff
@@ -1518,7 +1518,7 @@ namespace BranchInputManager {
 		int NumNodes; // Number of Nodes from NodeInputManager
 
 		Branch( BCount ).Name = Alphas( 1 );
-		Branch( BCount ).MaxFlowRate = Numbers( 1 );
+//		Branch( BCount ).MaxFlowRate = Numbers( 1 );
```

Comment:
@mjwitte @Myoldmopar Do you want this to come out like I did with the other deprecated Branch fields on this PR?